docs/demo/test_ok.py

```diff
@@ -1,18 +1,18 @@
-from __future__ import annotations
+from __future__ import annotations  # noqa: INP001
```

Comment:
If this is in `docs/`, it's intended for end-user consumption, right? Not just for maintainers? If so, I don't think we should add annotations here to appease our linter. I think we should tell the linter to ignore this file instead.

Reply:
I don't agree. Why should not we lint documentation files same as the rest?

Reply:
Because it makes the code harder to read by adding annotations that most of our users won't recognize or care about, which makes it function worse as documentation